pkg/event/target/webhook.go

```diff
@@ -209,14 +219,20 @@ func NewWebhookTarget(id string, args WebhookArgs, doneCh <-chan struct{}, logge
 	var store Store
 
 	target := &WebhookTarget{
-		id:   event.TargetID{ID: id, Name: "webhook"},
-		args: args,
-		httpClient: &http.Client{
-			Transport: transport,
-		},
+		id:         event.TargetID{ID: id, Name: "webhook"},
+		args:       args,
 		loggerOnce: loggerOnce,
 	}
 
+	if target.args.ClientCert != "" && target.args.ClientKey != "" {
+		cert, err := tls.LoadX509KeyPair(target.args.ClientCert, target.args.ClientKey)
```

Comment:
Look at the implementation of 

```
c, err = certs.New(target.args.ClientCert, target.args.ClientKey, tls.LoadX509KeyPair)
```

and add a helper for the `(c *Certs) GetClientCertifcate` matching this implementation

```
    GetClientCertificate func(*CertificateRequestInfo) (*Certificate, error) // Go 1.8
```

Reply:
Thanks for pointing this out.